src/main/java/com/teragrep/cfe_39/consumers/kafka/DatabaseOutput.java

```diff
@@ -0,0 +1,394 @@
+
+    public static final String ANSI_RESET = "\u001B[0m";
+    public static final String ANSI_GREEN = "\u001B[32m";
+    public static final String ANSI_BLUE = "\u001B[34m";
+    private SyslogAvroWriter syslogAvroWriter;
+    //    private final long minimumFreeSpace; // TODO: Check if needs to be implemented to the code.
+    private final long maximumFileSize;
+    private final WritableQueue writableQueue;
+    private final ByteBuffer sourceConcatenationBuffer;
+    private final SDVector teragrepStreamName;
+    private final SDVector teragrepDirectory;
+    private final SDVector eventNodeSourceSource;
+    private final SDVector eventNodeRelaySource;
+    private final SDVector eventNodeSourceSourceModule;
+    private final SDVector eventNodeRelaySourceModule;
+    private final SDVector eventNodeSourceHostname;
+    private final SDVector eventNodeRelayHostname;
+    private final SDVector originHostname;
+    private File syslogFile;
+    private final Config config;
+    //    private long approximatedSize; // FIXME: Not working properly when flush() is not used after append in the AVRO-file.
+    private long epochMicros_last;
+
+    DatabaseOutput(
+            Config config,
+            String table,
+            DurationStatistics durationStatistics,
+            TopicCounter topicCounter
+    ) {
+        this.config = config;
+        this.table = table;
+        this.durationStatistics = durationStatistics;
+        this.topicCounter = topicCounter;
+//        this.minimumFreeSpace = 32000000; // TODO: CHECK RIGHT VALUE FOR minimumFreeSpace
+        this.maximumFileSize = config.getMaximumFileSize(); // Maximum file size should be 64M (64000000). 60800000 is 95% of 64M which should be a good approximation point.
+
+        // queueDirectory and queueNamePrefix are only used for temporarily storing the AVRO-serialized files before committing them to HDFS when the file size reaches the threshold (or all records are processed).
+        this.writableQueue = new WritableQueue(
+                config.getQueueDirectory()
+        );
+
+        this.sourceConcatenationBuffer = ByteBuffer.allocateDirect(256*1024);
+        teragrepStreamName = new SDVector("teragrep@48577","streamname");
+        teragrepDirectory = new SDVector("teragrep@48577","directory");
+        this.eventNodeSourceSource = new SDVector("event_node_source@48577","source");
+        this.eventNodeRelaySource = new SDVector("event_node_relay@48577","source");
+        this.eventNodeSourceSourceModule = new SDVector("event_node_source@48577","source_module");
+        this.eventNodeRelaySourceModule = new SDVector("event_node_relay@48577","source_module");
+        this.eventNodeSourceHostname = new SDVector("event_node_source@48577","hostname");
+        this.eventNodeRelayHostname = new SDVector("event_node_relay@48577","hostname");
+        this.originHostname = new SDVector("origin@48577","hostname");
+//        this.approximatedSize = 0;
```

Comment:
Don't comment out code

Reply:
Solved in commit 022832c8b423efc7466b7aead56cc317a45a0002